spec/money_spec.rb

```diff
@@ -313,7 +313,7 @@
       expect(moneys[2].cents).to eq(33)
     end
 
-    specify "#allocate requires total to be less then 1" do
+    specify "#allocate requires total to be less than 1" do
```

Comment:
Your right, although sometimes he's a bit of a lose cannon. 
Alot of times I'm not sure he realizes how much affect he has on others. 
Weather he knows it or not, I can't be sure. 
Its definitaly wierd.
[Their, I said it](http://theoatmeal.com/comics/misspelling)